config/images/app-interface-push-images.Containerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest
```

Comment:
Isn't this file supposed to be in downstream? Also, is it tested somehow, where is it used?

Reply:
Yup - this file doesn't belong here